README.rst

```diff
@@ -156,7 +156,7 @@ or with python version 3:
 
 .. code:: console
 
-  python -c 'import urllib.request; print(urllib.request.urlopen("https://bootstrap.saltstack.com").read())' > bootstrap-salt.sh
+  python -c 'import urllib.request; print(urllib.request.urlopen("https://bootstrap.saltstack.com").read().decode("ascii"))' > bootstrap-salt.sh
```

Comment:
How about to change binary name to `python3`? Most distributions provide it like that.

Reply:
I'm ok with it, if it is more idiomatic.
python links to python3, which links to python3.7 on debian buster